lib/u3d/commands.rb

```diff
@@ -110,6 +110,7 @@ def download(args: [], options: {})
 
         unity = check_unity_presence(version: version)
         return unless enforce_setup_coherence(packages, options, unity)
+        return if packages.empty?
```

Comment:
Just looked at download_modules and I'm not sure this is where we want to parse empty packages as download packages. In my opinion this should be done in the `enforce_setup_coherence`, as it is precisely an operation that enforces make sure the request setup is correct. In any case, interpreting no packages as download all packages should be done before the `enforce_setup_coherence`. If not, we lose an important part of its functionnality and we create counter intuitive behaviours (precisely what I encountered yesterday), where when trying to install the exact setup you have, it filters out all your installed packages, and try to download all the packages, including the one you already have installed.